packages/valory/services/hello_world/service.yaml

```diff
@@ -7,43 +7,28 @@ license: Apache-2.0
 fingerprint:
   README.md: bafybeiapubcoersqnsnh3acia5hd7otzt7kjxekr6gkbrlumv6tkajl6jm
 fingerprint_ignore_patterns: []
-agent: valory/hello_world:0.1.0:bafybeihnvnab6qjgdbdm5dzumifs7junppbwokodx5uiwsz4vtnfyccnnm
+agent: valory/hello_world:0.1.0:bafybeiforw4w2w2jug57za6t7fhh3fryclpevp4kptajphndokogyr26nu
 number_of_agents: 4
 ---
-extra:
-  benchmark_persistence_params:
-    args: &id001
-      log_dir: /benchmarks
 public_id: valory/hello_world_abci:0.1.0
 type: skill
-0:
-  models:
-    params:
-      args:
-        hello_world_message: ${SERVICE_HELLO_WORLD_HELLO_WORLD_STRING:str:HELLO_WORLD!}
-    benchmark_tool:
-      args: *id001
-1:
-  models:
-    params:
-      args:
-        hello_world_message: ${SERVICE_HELLO_WORLD_HELLO_WORLD_STRING:str:HELLO_WORLD!}
-    benchmark_tool:
-      args: *id001
-2:
-  models:
-    params:
-      args:
-        hello_world_message: ${SERVICE_HELLO_WORLD_HELLO_WORLD_STRING:str:HELLO_WORLD!}
-    benchmark_tool:
-      args: *id001
-3:
-  models:
-    params:
-      args:
-        hello_world_message: ${SERVICE_HELLO_WORLD_HELLO_WORLD_STRING:str:HELLO_WORLD!}
-    benchmark_tool:
-      args: *id001
+models:
```

Comment:
Why this change? The documentation is aligned with having multiple overrides on this file as an example.

Reply:
Ahh I see, I'll revert this. 

However wouldn't it make more sense if the overrides could be different for the agents?

Reply:
Fixed on e0b8b8d528a7bc73146bdbc7c8e427b7f8cbca78.